certified-connectors/Cognite Data Fusion/apiProperties.json

```diff
@@ -7,29 +7,167 @@
       "token": {
         "type": "oauthSetting",
         "oAuthSettings": {
-          "identityProvider": "oauth2",
+          "identityProvider": "oauth2generic",
           "clientId": "PLACEHOLDER_CLIENTID",
-          "scopes": ["https://bluefield.cognitedata.com/.default"],
+          "scopes": ["{Scope}"],
           "redirectMode": "Global",
           "redirectUrl": "https://global.consent.azure-apim.net/redirect",
```

Comment:
As I mentioned earlier, could you kindly remove this redirectUrl completely, as this is not needed to add here and change redirectMode value from "Global" to "GlobalPerConnector".  Please refer this link. https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-5-submit-your-connector-for-deployment

Reply:
When I change redirectMode, I get this:

> Value is not accepted. Valid values: "Global", "Direct".

Reply:
Do I leave `redirect_uri={redirectUrl}` in my templates? or just remove them?